### PR TITLE
Refactor aapp runner

### PR DESCRIPTION
**Pull request by adybbroe**
- [ ] Closes #xxxx 
 - [ ] Tests added 
 - [x] Tests passed: Passes ``pytest`` 
 - [ ] Passes ``flake8`` 
 - [ ] Fully documented

### CODE REVIEW

File: aapp_runner/helper_functions.py [hunk 1]
```diff
@@ -25,10 +26,22 @@
...
 
+def get_local_ips():
+    inet_addrs = [netifaces.ifaddresses(iface).get(netifaces.AF_INET)
+                  for iface in netifaces.interfaces()]
+    ips = []
+    for addr in inet_addrs:
+        if addr is not None:
+            for add in addr:
+                ips.append(add['addr'])
+    return ips
```
Comment (mraspaud) [inline aapp_runner/helper_functions.py]: Could we import the posttroll version instead? https://github.com/pytroll/posttroll/blob/67b8f3d4df27cbb08aaa4baa681acac73d98bd99/posttroll/address_receiver.py#L60-L68

Comment (adybbroe) [reply aapp_runner/helper_functions.py]: Yes, good point!
Optimally this function should be moved into `__init__.py` or a `utils.py` file in Posttroll. Should I make an issue for that do you think?
Now, I would have to do
`from posttroll.address_receiver import get_local_ips`
I mean it is not intuitive that I must import this from the `address_receiver` of Posttroll...

**Comment by adybbroe** [comment]
Think I would like to leave it here. There is much more one could do, but the runner works and runs fine here at SMHI now, and I am not sure I can devote much more time to it now.

**Comment by mraspaud** [comment]
Feel free to merge this if you are happy with it. Maybe testing/refactoring should be done in a concerted effort, for example at a PCW?

**Pull request merged**